spec/providers/atmosphere/local_pdp_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+
+describe Atmosphere::LocalPdp do
+  context '#can_manage?' do
+    let(:user) { build(:user, id: 1) }
+    subject { Atmosphere::LocalPdp.new(user) }
+
+    it 'allows to manage owned appliance types' do
+      at = build(:appliance_type, user_id: user.id)
```

Comment:
There's a special exception for services whose `user_id` attribute matches the requestor's ID (these are considered "owned" by the given user). This is also why the `User` object was built with a specific ID (but I've reworked it anyway).